configuration/analyzer_config.json

```diff
@@ -957,6 +957,21 @@
       }
     }
   },
+  "ELF_Info": {
+    "type": "file",
+    "python_module": "elf_info.ELFInfo",
+    "description": "static ELF analysis",
+    "disabled": false,
+    "external_service": false,
+    "leaks_info": false,
+    "supported_filetypes": ["application/x-sharedlib", "application/x-elf"],
```

Comment:
please add `"application/x-executable"` too. 

And please add that new mimetype you added (`"application/x-elf"`) in the other places where we have `"application/x-executable"` so we can handle both cases

Reply:
This is something I wanted to ask. Any files I uploaded showed up as `application/x-sharedlib`. I added `application/x-elf` just for theoretical completeness. Should I remove it?

Reply:
no keep that, just add also mine there and everywhere we use filters for ELF files. The mimetype thing is a jungle and there could be a lot of other in the wild still valid. So more is better